diff -up gupnp-igd-0.2.5/configure.ac.omv~ gupnp-igd-0.2.5/configure.ac
--- gupnp-igd-0.2.5/configure.ac.omv~	2019-02-12 02:08:24.809684040 +0100
+++ gupnp-igd-0.2.5/configure.ac	2019-02-12 02:08:24.802684174 +0100
@@ -11,7 +11,7 @@ AC_STDC_HEADERS
 AC_PROG_LIBTOOL
 AC_FUNC_MMAP
 
-PKG_CHECK_MODULES(LIBGUPNP, glib-2.0 >= 2.26 gobject-2.0 >= 2.26 gssdp-1.0 gupnp-1.0 >= 0.18 gthread-2.0)
+PKG_CHECK_MODULES(LIBGUPNP, glib-2.0 >= 2.26 gobject-2.0 >= 2.26 gssdp-1.2 gupnp-1.2 gthread-2.0)
 
 # glib-genmarshal
 GLIB_GENMARSHAL=`pkg-config --variable=glib_genmarshal glib-2.0`
diff -up gupnp-igd-0.2.5/gupnp-igd-1.0.pc.in.omv~ gupnp-igd-0.2.5/gupnp-igd-1.0.pc.in
--- gupnp-igd-0.2.5/gupnp-igd-1.0.pc.in.omv~	2019-02-12 02:08:19.691782418 +0100
+++ gupnp-igd-0.2.5/gupnp-igd-1.0.pc.in	2019-02-12 02:08:22.273732608 +0100
@@ -5,7 +5,7 @@ includedir=@includedir@
 
 Name: gupnp-igd-1.0
 Description: GUPnP Simple IGD library
-Requires: gupnp-1.0
+Requires: gupnp-1.2
 Version: @VERSION@
 Libs: -L${libdir} -lgupnp-igd-1.0
 Cflags: -I${includedir}/gupnp-igd-1.0
diff -up gupnp-igd-0.2.5/gupnp-igd-1.0-uninstalled.pc.in.omv~ gupnp-igd-0.2.5/gupnp-igd-1.0-uninstalled.pc.in
--- gupnp-igd-0.2.5/gupnp-igd-1.0-uninstalled.pc.in.omv~	2019-02-12 02:07:13.021200263 +0100
+++ gupnp-igd-0.2.5/gupnp-igd-1.0-uninstalled.pc.in	2019-02-12 02:07:17.148104820 +0100
@@ -5,7 +5,7 @@ includedir=${pcfiledir}/
 
 Name: gupnp-igd-1.0
 Description: GUPnP Simple IGD library
-Requires: gupnp-1.0
+Requires: gupnp-1.2
 Version: @VERSION@
 Libs: ${libdir}/libgupnp-av-1.0.la
 Cflags: -I${includedir}
diff -up gupnp-igd-0.2.5/libgupnp-igd/GUPnPIgd-1.0.gir.omv~ gupnp-igd-0.2.5/libgupnp-igd/GUPnPIgd-1.0.gir
--- gupnp-igd-0.2.5/libgupnp-igd/GUPnPIgd-1.0.gir.omv~	2019-02-12 02:07:54.325291434 +0100
+++ gupnp-igd-0.2.5/libgupnp-igd/GUPnPIgd-1.0.gir	2019-02-12 02:10:00.753087582 +0100
@@ -7,7 +7,7 @@ and/or use gtk-doc annotations.  -->
             xmlns:c="http://www.gtk.org/introspection/c/1.0"
             xmlns:glib="http://www.gtk.org/introspection/glib/1.0">
   <include name="GObject" version="2.0"/>
-  <package name="gupnp-1.0"/>
+  <package name="gupnp-1.2"/>
   <namespace name="GUPnPIgd"
              version="1.0"
              shared-library="libgupnp-igd-1.0.so.4"
diff -up gupnp-igd-0.2.5/libgupnp-igd/Makefile.am.omv~ gupnp-igd-0.2.5/libgupnp-igd/Makefile.am
--- gupnp-igd-0.2.5/libgupnp-igd/Makefile.am.omv~	2019-02-12 02:08:24.782684556 +0100
+++ gupnp-igd-0.2.5/libgupnp-igd/Makefile.am	2019-02-12 02:08:24.781684575 +0100
@@ -60,7 +60,7 @@ GUPnPIgd_1_0_gir_VERSION = 1.0
 GUPnPIgd_1_0_gir_LIBS = $(lib_LTLIBRARIES)
 GUPnPIgd_1_0_gir_FILES = $(libgupnp_igd_1_0_la_SOURCES) $(libgupnp_igd_inc_HEADERS)
 GUPnPIgd_1_0_gir_INCLUDES=GObject-2.0
-GUPnPIgd_1_0_gir_PACKAGES=gupnp-1.0
+GUPnPIgd_1_0_gir_PACKAGES=gupnp-1.2
 GUPnPIgd_1_0_gir_CFLAGS=-I$(srcdir) -I$(top_srcdir)
 
 if HAVE_INTROSPECTION
